dspace/config/item-submission.xml

```diff
@@ -119,14 +119,15 @@
             <processing-class>org.dspace.app.rest.submit.step.CCLicenseStep</processing-class>
 	       <type>cclicense</type> </step-definition> -->
 
-        <!--Step will be to Check for potential duplicate -->
-        <!-- <step-definition id="detect-duplicate"> <heading>submit.progressbar.detect.duplicate</heading>
-            <processing-class>org.dspace.submit.step.DetectPotentialDuplicate</processing-class> 
-            <type>duplicate</type> </step-definition> -->
-
-        <!--Step will be to Verify/Review everything -->
-        <!-- <step-definition id="verify"> <heading>submit.progressbar.verify</heading> <processing-class>org.dspace.submit.step.VerifyStep</processing-class>
-            <type>verify</type> </step-definition> -->
+        <!--Step will be to enrich the current submission querying external providers or processing the uploaded file -->
+        <!-- Uncomment this step to enrich the current submission using information extracted 
+            from uploaded files or metadata. Please check also config/spring/api/step-processing-listener.xml
+            for further configuration -->
```

Comment:
Just a note which we may want to add to this configuration.  It looks like, currently, this step is only triggered during either:
* An **upload** (e.g. extract metadata from a BibTex file)
* A **save** of the form. (e.g. entering a PubMed ID only triggers a lookup if you click "Save" or "Save for later")

Reply:
done. I have also mentioned the autosave feature available on the angular side that can be used with that to make the process more user friendly